src/views/tutorial-view/server.ts

```diff
@@ -166,9 +179,11 @@ export async function getTutorialPagePaths(): Promise<TutorialPagePaths[]> {
 		 * @TODO once we implement the `theme` query option, remove the theme filtering
 		 * https://app.asana.com/0/1201903760348480/1201932088801131/f
 		 */
-		const shouldBuildTutorialPath =
-			getIsBetaProduct(productSlugFromCollection as LearnProductSlug) &&
-			productSlugFromCollection === collection.theme
+		const productSlug = normalizeProductSlugForDevDot(productSlugFromCollection)
+		const isBetaProduct = getIsBetaProduct(productSlug)
+		const isCloud = productSlugFromCollection == 'cloud'
+		const themeMatches = productSlug === collection.theme
+		const shouldBuildTutorialPath = isBetaProduct && (isCloud || themeMatches)
```

Comment:
Adjusted the logic here slightly to get `hcp` content to render (despite `collection.theme` not matching for `cloud` collections coming from the Learn API).